### DISCUSSION TITLE
MQTT payload object meanings help

### DISCUSSION THREAD

**Discussion by rocket59** [blakeblackshear/frigate#9351] [Ask A Question] [answered]
I'm trying to figure out what some of these payload objects mean, can someone help? (the docs are a bit light on this)
- top_score vs score
- what's the sub label for?
- what's a region?
- is box is the size of the object bounding box? What does the box ratio mean?

Thanks! The MQTT support is great - I'm coming to grips with filtering the messages into meaningful alerts I can send to my phone!

**Comment by NickM-27** [comment, marked as answer]
> top_score vs score

Top score is the highest median score that the object has achieved, score is the score of the object at that current moment in time. 
> what's the sub label for?

Sub labels are for the name of people, type of vehicle (like ups, Amazon, etc) but they all require an external service like doubletake face recognition 

> what's a region?

Region is the portion of the image that was sent to object detection

> is box is the size of the object bounding box? What does the box ratio mean?

Box ratio is the ratio of the boxes width to its height

**Reply by rocket59** [reply]
Thank you!
